src/test/java/tests/LoginTest.java

```diff
@@ -35,4 +36,53 @@ public void TC002() {
 
         Assert.assertEquals(actualErrorMsg, expectedErrorMsg, "Error message is incorrect.");
     }
+
+    @Test(description = "User cannot log into Railway with invalid password")
+    public void TC003() {
+        homePage.gotoLoginPage();
+
+        String invalidPassword = DataHelper.getRandomText();
+
+        loginPage.login(Constant.USERNAME, invalidPassword);
+
+        String expectedErrorMsg = "There was a problem with your login and/or errors exist in your form.";
+        String actualErrorMsg = loginPage.getErrorMsg();
+
+        Assert.assertEquals(actualErrorMsg, expectedErrorMsg, "Error message is incorrect.");
```

Comment:
why two verify point with only different failed message?

Reply:
Thanks for your review. I just have updated my code, please review it for me.